_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+
+{% image blog/blog-ecommerce-barcamp-4.jpg 45% right %}The orga team{% endimage %}
+
+Another interesting session was a guided discussion about [Magento 2](https://magento.com/developers/magento2). Björn Jacob and Tobias Vogt explained in an entertaining fashion the new functio set in frequent interplay with the strongly involved audience. Over the whole session length, the enhancement of the productivity and performance improvement as well as the overall stability of the new major relase were questioned very often. The consolidated experience of the session room concluded that serveral good architectural decisions for this release were made, e.g.: [less](http://lesscss.org/) compiler, modes (default, env, prod), stricter separation of layout and logic via templates and modules, dependency injection, service contracts, XML schema definition for config files. The software project has become much more open and community driven. Another observation was that version 2.1 might be able to offer a competive performance.
+
+The last track on Friday was offered by Andreas Ziethen, who humbly claims to be the senior apprentice at his company. In fact he is the highly likeable head of RND that flawlessly proved his comprehensive knowledge about deployment automation in front of a full room. The encouraging talk focused on the usage of [GitLab](https://www.gitlab.com) for deployment and [Rundeck](http://rundeck.org) for job scheduling and runbook automation. Today, the interplay of both technologies is already at a very mature quality level and typical tasks like staging and conditional build steps can be represented within versioned job descriptions. If you have a small shop project with a minimal set of requirements for your CI and CD pipeline setup the discussed solution might be competitive with [Jenkins](https://jenkins-ci.org), [Bamboo](https://confluence.atlassian.com/bamboo) or [TeamCity](https://www.jetbrains.com/teamcity/).
+
+On Saturday there were only three time slots before the closing speech. 
+
```

Comment:
See above comment on skipping the speaker names as the readers might not be familiar with them.